content/en/docs/event-sources/gvisor/_index.md

```diff
@@ -0,0 +1,171 @@
+---
+title: gVisor
+description: 'gVisor support in Falco'
+linktitle: gVisor
+weight: 80
+---
+
+Falco can work with **[gVisor](https://gvisor.dev/)**.
+
+gVisor, quoting the [official documentation](https://gvisor.dev/docs), is an application kernel that provides an **additional layer of isolation** between running applications and the host operating system. It delivers an additional security boundary for containers by **intercepting and monitoring workload runtime instructions in user space** before they can reach the underlying host.
+
+## How Falco and gVisor work together
+
+When running containers with gVisor, there are several components that interact with our workload:
+
+![](images/gvisor-01.png)
+
+The Sentry is the gVisor component that implements all the application kernel functionalities. Whenever a syscall gets executed inside the sandboxed application, the Sentry will manage it as usual, plus it will send a message to Falco through a UDS (Unix Domain Socket). 
+
+Messages are serialized through Protocol Buffers so that gVisor and Falco can communicate even if they are written in different programming languages.
+
+![](images/gvisor-02.png)
+
+## Setup gVisor Docker sandbox monitoring with Falco
+
+First, [install **Falco**](https://falco.org/docs/getting-started/installation/) 0.371 or above and [install the **gVisor runsc tool**](https://gvisor.dev/docs/user_guide/install/) 20221122.0 or above:
+
+- Check the Falco version is `>= 0.33.1`:
+  ```bash
+  falco --version
+  ```
+- Check the `runsc` version is `release-20221122.0`:
+  ```bash
+  runsc --version
+  ```
```

Comment:
```suggestion
First, [install **Falco**](https://falco.org/docs/getting-started/installation/) and [install the **gVisor runsc tool**](https://gvisor.dev/docs/user_guide/install/). Any version of runsc released in 2023 or later is compatible with Falco.
```

Let's simplify this. Back then we had to specify exact versions because we were using cutting edge builds, now these changes are more stable.